qualang_tools/bakery/xeb.py

```diff
@@ -1,5 +1,5 @@
 from dataclasses import dataclass
-from typing import List, Callable, Union
```

Comment:
Why are these removed? They are not used?
If I remember correctly, @arthurostrauss added them because of an issue Caltech had...

Reply:
Correct, those aliases worked and were used to assert the type of the expected arguments in all the methods. Caltech had the issue when the type was directly specified (list instead of List for example).

Reply:
that makes sense, but they are not used in this file. I've added linter so deleted the unused imports

Reply:
I think that they were used in the type validation, not?

Reply:
It is used in the bakery file, it is unneeded here.